projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip-base.scss

```diff
@@ -0,0 +1,238 @@
+@import '../../../../angular-core/styles/common/variants';
```

Comment:
Why was this extracted to its own file?
The intention going forward is to remove these separate styles folders and move the sass code into the *.component.scss files.